doc/ref_model/chapters/chapter02.md

```diff
@@ -5,7 +5,7 @@
 - [2.1 Workloads Collateral](#21-workloads-collateral)
 - [2.2 Use cases](#22-use-cases)
 - [2.3 Analysis](#23-analysis)
-- [2.4 Profiles & Flavours](#24-profiles--flavours)
+- [2.4 Profiles, Profile-Extensions & Flavours](#24-profiles--flavours)
```

Comment:
Profile-Extensions -> Profile Extensions? Also in text below.

Reply:
@petorre Thanks - Done.